pkg/cmd/job/job_kill.go

```diff
@@ -0,0 +1,61 @@
+package job
+
+import (
+	"fmt"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/spf13/cobra"
+)
+
+// newCmdJobKill kills a run of a job
+func newCmdJobKill(ctx api.Context) *cobra.Command {
+	var all bool
+	cmd := &cobra.Command{
+		Use:   "kill <job-id> [OPTIONS]",
+		Short: "kill a job",
+		Args:  cobra.MaximumNArgs(2),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 2 && all == true {
+				return fmt.Errorf("cannot accept both a run-id and the --all option")
+			}
+			if len(args) == 1 && all == false {
+				return fmt.Errorf("run-id must be specified or --all option must be set")
+			}
+			if len(args) == 0 {
+				return fmt.Errorf("cannot run `dcos job kill` without any options")
+			}
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client, err := metronomeClient(ctx)
+			if err != nil {
+				return err
+			}
+
+			var deadpool []string
+			if len(args) == 1 && all == true {
+				runs, err := client.Runs(args[0])
```

Comment:
How the runs from here are different from the ones in `Job.ActiveRuns`?

Reply:
We could get them through that field too but I figured I would use the extra endpoint the API provides for it.

Reply:
ok, I was just curious if there were any difference between the 2